### PR TITLE
Fix Formatting issues

### LINKED ISSUE #28: Update doc for backend security scheme
### Description:

This document contain multiple formatting issues
### Content Positioning in Documentation:

- Link:
https://mg.docs.wso2.com/en/3.2.0/how-tos/defining-a-backend-security-scheme/

- Heading (& Step):

- Any other reference:

---

### LINKED ISSUE #35: Few documentation issues in "installation prerequisites" doc
### Description:
https://mg.docs.wso2.com/en/latest/install-and-setup/install-on-vm/ has outdated content about APIM version and few rendering issues.

Rendering issues:

image

Mentions about APIM 3.0.0 and should be updated.

image

### Content Positioning in Documentation:

- Link:

- Heading (& Step):

- Any other reference:

---

### LINKED ISSUE #47: Incorrect sub section heading in "Quick Start Guide - Binary"
### Description:
Following heading is incorrect.

docs-mg

### Content Positioning in Documentation:
- Link:
https://mg.docs.wso2.com/en/latest/getting-started/quick-start-guide/quick-start-guide-binary/#step-3-invoke-the-sample-api

- Heading (& Step):
Quick Start Guide - Binary

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
### Purpose

Rename "howtos->Adding Extension points" to "howtos->Adding Extensions"
Rename "howtos->support-custom-claims-mapping" to "howtos->Custom Claims Mapping"
Fixes following issues.

### Issues

Fixes #47 
Fixes #35
Fixes #28

---
#### Maintainers: Check before merge
- [ ] Manual verification

### CODE REVIEW

File: en/docs/install-and-setup/install-on-vm.md [hunk 1]
```diff
@@ -7,55 +7,56 @@ Follow the instructions below to install the WSO2 API Microgateway related binar
...
+1.  Download the [WSO2 API Microgateway toolkit distribution](https://wso2.com/api-management/api-microgateway/).
 
-2.  Download the [WSO2 API Microgateway toolkit distribution](https://wso2.com/api-management/api-microgateway/) .
-
-3.  Extract the archive file to a dedicated directory for the WSO2 API Microgateway **Toolkit** , which will hereafter be referred to as `           <MGW-TK_HOME>          ` .
-
-4.  Append the full path of the `           /bin          ` folder of the extracted WSO2 API Microgateway toolkit distribution to the PATH environment variable.
+2.  Extract the archive file to a dedicated directory for the WSO2 API Microgateway Toolkit, which will hereafter be referred to as `<MGW-TK_HOME>`.
+    
+3.  Append the full path of the `/bin` folder of the extracted WSO2 API Microgateway toolkit distribution to the PATH environment variable.
```
Comment (praminda) [inline en/docs/install-and-setup/install-on-vm.md:14]: better if we don't use folder and directory terms interchangeably in the docs. Shall we use directory?
```suggestion
3.  Append the full path of the `/bin` directory of the extracted WSO2 API Microgateway toolkit distribution to the PATH environment variable.
```

File: en/docs/install-and-setup/install-on-vm.md [hunk 2]
```diff
@@ -7,55 +7,56 @@ Follow the instructions below to install the WSO2 API Microgateway related binar
...
-        4.  Navigate to the path system variable and click **Edit.**
-
-            ![](attachments/141247030/141247031.png)
-
-        5.  Click **New,** add the path of the Microgateway Toolkit `                  bin                 ` folder that you copied previously and click **OK** three times to save the newly added system variable.
-            ![](attachments/141247030/141247032.png)
-
-5.  Optionally, [configure WSO2 API Microgateway Toolkit with WSO2 API Manager 3.0.0]({{base_path}}/install-and-setup/configuration-for-wso2-api-manager/#configuring-the-microgateway-31x-toolkit) .
+    
+        1.  Copy the path to the Microgateway Toolkit `bin` folder (`<path-to-wso2am-micro-gw-toolkit>\bin`).
```
Comment (praminda) [inline en/docs/install-and-setup/install-on-vm.md:38]:
```suggestion
        1.  Copy the path to the Microgateway Toolkit `bin` directory (`<path-to-wso2am-micro-gw-toolkit>\bin`).
```

File: en/docs/install-and-setup/install-on-vm.md [hunk 3]
```diff
@@ -7,55 +7,56 @@ Follow the instructions below to install the WSO2 API Microgateway related binar
...
+        
+        3.  Navigate to the **Advanced** tab and click **Environment Variables**.
+        
+            ![]({{base_path}}/assets/img/install-and-setup/Environment-variables.png)
+        
+        4.  Navigate to the path system variable and click **Edit**.
+        
+            ![]({{base_path}}/assets/img/install-and-setup/select-path-system-var.png)    
+        
+        5.  Click **New,** add the path of the Microgateway Toolkit `bin` folder that you copied previously and click **OK** three times to save the newly added system variable.
```
Comment (praminda) [inline en/docs/install-and-setup/install-on-vm.md:52]:
```suggestion
        5.  Click **New,** add the path of the Microgateway Toolkit `bin` directory that you copied previously and click **OK** three times to save the newly added system variable.
```

File: en/docs/install-and-setup/install-on-vm.md [hunk 4]
```diff
@@ -7,55 +7,56 @@ Follow the instructions below to install the WSO2 API Microgateway related binar
...
+    
+        ``` java tab="Format"
+        setx "%path%;<extracted_toolkit_location>/bin"
         ```
-
-            -   Set the Microgateway toolkit distribution path using the UI in Windows.
-
-            !!! note
+    
+    2.  Set the Microgateway toolkit distribution path using the UI in Windows.
```
Comment (praminda) [inline en/docs/install-and-setup/install-on-vm.md:33]: I'm wondering if we need to provide this level of details instructions to add an item to `PATH` variable

File: en/docs/install-and-setup/install-on-vm.md [hunk 5]
```diff
@@ -67,40 +68,34 @@ Follow the instructions below to install the WSO2 API Microgateway related binar
...
-2.  Log in to the command line (Terminal on Mac).
-
-3.  Download the [WSO2 API Microgateway distribution](https://wso2.com/api-management/api-microgateway/) based on your OS and extract it.
+2.  Download the [WSO2 API Microgateway distribution](https://wso2.com/api-management/api-microgateway/) based on your OS and extract it.
 
-4.  Extract the archive file to a dedicated directory for the WSO2 API Microgateway **Runtime** , which will hereafter be referred to as `           <MGW_HOME>          ` .
+3.  Extract the archive file to a dedicated directory for the WSO2 API Microgateway **Runtime**, which will hereafter be referred to as `<MGW_HOME>`.
 
-5.  Append the full path of the `           /bin          ` folder of the extracted WSO2 API Microgateway distribution to the PATH environment variable.
+4.  Append the full path of the `/bin` folder of the extracted WSO2 API Microgateway distribution to the PATH environment variable.
```
Comment (praminda) [inline en/docs/install-and-setup/install-on-vm.md:75]:
```suggestion
4.  Append the full path of the `/bin` directory of the extracted WSO2 API Microgateway distribution to the PATH environment variable.
```

File: en/docs/install-and-setup/install-on-vm.md (hunk 1 above)
Comment (AmaliMatharaarachchi) [reply en/docs/install-and-setup/install-on-vm.md]: Fixed https://github.com/wso2/docs-mg/pull/54/commits/22e1005ff35ccb1e2f1679aeeaba27e6d514bf6c

File: en/docs/install-and-setup/install-on-vm.md (hunk 3 above)
Comment (AmaliMatharaarachchi) [reply en/docs/install-and-setup/install-on-vm.md]: Fixed https://github.com/wso2/docs-mg/pull/54/commits/22e1005ff35ccb1e2f1679aeeaba27e6d514bf6c

File: en/docs/install-and-setup/install-on-vm.md (hunk 4 above)
Comment (AmaliMatharaarachchi) [reply en/docs/install-and-setup/install-on-vm.md:34]: fixed https://github.com/wso2/docs-mg/pull/54/commits/22e1005ff35ccb1e2f1679aeeaba27e6d514bf6c

File: en/docs/install-and-setup/install-on-vm.md (hunk 5 above)
Comment (AmaliMatharaarachchi) [reply en/docs/install-and-setup/install-on-vm.md]: fixed https://github.com/wso2/docs-mg/pull/54/commits/22e1005ff35ccb1e2f1679aeeaba27e6d514bf6c

File: en/docs/install-and-setup/install-on-vm.md [hunk 6]
```diff
@@ -7,55 +7,57 @@ Follow the instructions below to install the WSO2 API Microgateway related binar
...
+        
+        5.  Click **New,** add the path of the Microgateway Toolkit `bin` directory that you copied previously and click **OK** three times to save the newly added system variable.
+    
+            ![]({{base_path}}/assets/img/install-and-setup/add-new-variable.png)
+    </details>    
+4.  Optionally, [configure WSO2 API Microgateway Toolkit with WSO2 API Manager 3.2.0]({{base_path}}/install-and-setup/configuration-for-wso2-api-manager/#configuring-the-microgateway-32x-toolkit) .
 
     !!! note
-        This is only applicable if you are working with WSO2 API Manager 3.0.0.
+        This is only applicable if you are working with WSO2 API Manager 3.2.0.
```
Comment (shaniR) [inline en/docs/install-and-setup/install-on-vm.md:60]: what about the earlier versions?

File: en/docs/install-and-setup/install-on-vm.md [hunk 7]
```diff
@@ -67,40 +69,35 @@ Follow the instructions below to install the WSO2 API Microgateway related binar
...
 
-    -   **Linux and OS X**
-
-        **Example**
+    1.  Set the Microgateway toolkit distribution path using the following command in the terminal.
 
-        ``` java
-            export PATH=$PATH:<extracted_distribution_location>/bin
+        ``` java tab="Format"
+        setx "%path%;<extracted_distribution_location>/bin"
```
Comment (shaniR) [inline en/docs/install-and-setup/install-on-vm.md:93]: setx?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/install-and-setup/install-on-vm.md (hunk 7 above)
Comment (AmaliMatharaarachchi) [reply en/docs/install-and-setup/install-on-vm.md:93]: It's a windows command https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx

**Review by shaniR — APPROVED** [review]

**Pull request merged**